src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -1,10 +1,83 @@
-import React, { FC, HTMLAttributes, ReactNode, useMemo } from 'react'
-import styled, { css } from 'styled-components'
+import React, { ComponentPropsWithoutRef, FC, PropsWithChildren, useMemo } from 'react'
+import { tv } from 'tailwind-variants'
 
-import { Theme, useTheme } from '../../hooks/useTheme'
 import { FaExclamationCircleIcon, FaExclamationTriangleIcon } from '../Icon'
 
-import { useClassNames } from './useClassNames'
+const statusLabel = tv({
+  base: [
+    'shr-box-content',
+    'shr-font-bold',
+    'shr-inline-flex',
+    'shr-items-center',
+    'shr-justify-center',
+    'shr-gap-0.25',
+    'shr-px-0.5',
+    'shr-py-0.25',
+    'shr-whitespace-nowrap',
+    'shr-text-sm',
+    // ラベルが天地中央に揃わないため暫定対応
+    'shr-leading-[0]',
+    'shr-min-w-[3.5em]',
+    'shr-min-h-em',
+    'shr-border',
+    'shr-border-solid',
+  ],
+  variants: {
+    type: {
+      grey: ['contrast-more:shr-border-high-contrast'],
+      blue: ['shr-border-main', 'shr-text-link'],
```

Comment:
`TEXT_LINK` は元々なさそう。

```suggestion
      blue: ['shr-border-main'],
```

Reply:
ここはtext colorにmainを追加しました！

Reply:
ref: https://github.com/kufu/smarthr-ui/pull/4119/files#diff-47a4e14c995c4c3ce962ac60dced66783000d4721073c461b8bf9f30612ed5ceR135-R136